### DISCUSSION TITLE
D0D84442-1D8B-636C-90B3-606CA1CC8EFE

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#15] [General]
# ABU: 107 Piscium

This system focuses on the habitable world of Kelari, which along with three other worlds (Muhrashi, Chiloque, and Juiravan) share a common barycenter. This system should have been chaotic over a few millenia, but it held stable gravitationally by going near their common Lagrange points in space

https://celestia.mobi/addon?item=D0D84442-1D8B-636C-90B3-606CA1CC8EFE&lang=es

**Comment by zamoras14** [comment]
It doesn't work, when I search for 107 psc the planets don't appear

**Reply by DaveBowman2001** [reply]
Finally pinpointed the problem. It is highly likely that the error is caused by the planets addressing their parent star as `107 Piscium` instead of `107 Psc`. Should be fixed now by a patch update that is just published today as of this writing
